### DISCUSSION TITLE
联想词怎么自定义？

### DISCUSSION THREAD

**Discussion by captss** [imfuxiao/Hamster#442] [Q&A]
参照搜狗的联想词，但是报错了INFO E20231205 10:16:19.758740 2934202 entry_collector.cc:134] Encode failure:

**Comment by imfuxiao** [comment]
仓目前还不支持联想功能

**Comment by captss** [comment]
rime-ice.zip是怎么压缩的，为什么解压后再压缩，就加载不了了？

发自我的iPhone


------------------ 原始邮件 ------------------
发件人: morse.hsiao ***@***.***&gt;
发送时间: 2023年12月5日 11:46
收件人: imfuxiao/Hamster ***@***.***&gt;
抄送: captss ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [imfuxiao/Hamster] 联想词怎么自定义？ (Discussion #442)





 
仓目前还不支持联想功能

**Reply by imfuxiao** [reply]
应该是您的压缩方式有问题，根目录不应该包含文件夹。

**Comment by captss** [comment]
全选文件然后压缩可以。



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: morse.hsiao ***@***.***&gt;
发送时间: 2023年12月11日 10:33
收件人: imfuxiao/Hamster ***@***.***&gt;
抄送: captss ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [imfuxiao/Hamster] 联想词怎么自定义？ (Discussion #442)





 
应该是您的压缩方式有问题，根目录不应该包含文件夹。
